samples/Mvc.RenderViewToString/Program.cs

```diff
@@ -32,18 +35,26 @@ public static string RenderView(string customApplicationBasePath = null)
             // Add a custom service that is used in the view.
             services.AddSingleton<EmailReportGenerator>();
 
-            var provider = services.BuildServiceProvider();
-            var helper = provider.GetRequiredService<RazorViewToStringRenderer>();
+            var serviceProvider = services.BuildServiceProvider();
+            return serviceProvider.GetRequiredService<IServiceScopeFactory>();
+        }
 
-            var model = new EmailViewModel
+        public static Task<string> RenderViewAsync(IServiceScopeFactory scopeFactory)
```

Comment:
Can this name stay as `RenderView` and return `.Result`? The real async stuff happens in the helper right?